TestHelpers/API/TestDataHelpers.swift

```diff
@@ -3,33 +3,41 @@ import Foundation
 /// A type that wraps fixture data for use in mocking responses during tests.
 ///
 public enum TestDataHelpers {
-    /// The class used to determine the bundle to load files from.
-    public static var bundleClass: AnyClass?
+    /// The default bundle to try loading files from.
+    public static var defaultBundle: Bundle?
 
     /// Loads the data from the provided file.
-    public static func loadFromBundle(resource: String, extension: String) -> Data {
-        guard let bundleClass else {
-            fatalError("Class to determine test bundle from not set properly in the test case.")
+    public static func loadFromBundle(resource: String, extension: String, bundle: Bundle? = nil) -> Data {
+        let resolvedBundle: Bundle
+        switch bundle {
+        case .none:
+            guard let defaultBundle else {
+                fatalError("Default test data bundle from not set properly in the test case.")
+            }
+            resolvedBundle = defaultBundle
+        case let .some(bundle):
+            resolvedBundle = bundle
         }
```

Comment:
🎨 I think it's simpler to rewrite this as:
```suggestion
        let resolvedBundle = bundle ?? defaultBundle
        guard let resolvedBundle else {
            fatalError("Default test data bundle from not set properly in the test case.")
        }
```

Reply:
It is! Good call!